mirrord/cli/src/config.rs

```diff
@@ -217,3 +221,37 @@ pub(super) enum OperatorCommand {
         namespace: OperatorNamespace,
     },
 }
+
+#[derive(ValueEnum, Clone, Default, Debug)]
+pub enum Format {
+    #[default]
+    Json,
+}
+
+impl FromStr for Format {
```

Comment:
I don't think it's needed?

Reply:
https://github.com/clap-rs/clap/blob/master/examples/tutorial_derive/04_01_enum.rs